packages/react-components/src/button.tsx

```diff
@@ -124,6 +124,7 @@ const primaryStyles = css({
     backgroundColor: pine.rgb,
     borderColor: pine.rgb,
     boxShadow: 'none',
+    color: paper.rgb,
```

Comment:
why is it necessary?

Reply:
Without that, the text color on active buttons will be wrong (a darker one). Please, check https://trello.com/c/DR4BaNwq/1113-text-on-primary-green-buttons-goes-black-on-click-should-stay-white

Reply:
Yep, I can see what it tries to achieve, but could you reproduce that? For me it always stays white. Is it browser-specific because of certain user agent stylesheets or something?

Reply:
On master already always stays white, that is

Reply:
I can replicate it, yes.

Reply:
macOS - Brave browser: confirmed
Ubuntu - Chrome browser: confirmed

Which OS and browser are you using?

Reply:
I've just checked and it seems it comes from `Anchor` `resetStyles`, so it is only there for `buttonStyle primary` `Link`s